data_structures/stacks/infix_to_prefix_conversion.py

```diff
@@ -15,7 +15,7 @@
 """
 
 
-def infix_2_postfix(infix):
+def infix_2_postfix(infix: str) -> str:
```

Comment:
BEFORE we modify this function we need it to have doctests.  You can do that in a separate pull request and then we can circle back to this pull request AFTER the doctests have been merged.

Reply:
I'm reaching my PR limit so maybe I can commit changes directly here or can you help me review #10073 or #10012 first?

Reply:
I merged one of them that adds tests.  The other one deleted the work of others but perhaps it is the right thing to do.

Reply:
ok thanks i'll work on this

Reply:
@cclauss I added tests in #10095, you can help me look through.